test/settings/DynamoSettings-NewSettings.xml

```diff
@@ -40,6 +40,7 @@
   <BackupFiles>
     <string>C:\Users\user\Documents\dynamofile.dyn</string>
   </BackupFiles>
+  <TemplateFilePath>C:\ProgramData\Dynamo\Dynamo Core\templates\en-US</TemplateFilePath>
```

Comment:
@zeusongit It looks like now this property is serialized always? Although the value could be empty. Please confirm

Reply:
Hi @QilongTang  - I am not the author of the preference property, I assume it's a new addition. https://github.com/DynamoDS/Dynamo/blob/97e2d4651637ac20c235992167f0be3cadec5593/src/DynamoCore/Configuration/PreferenceSettings.cs#L949
The test picks it up as the number of default preferences coming from `PreferenceSettings.cs` was different then the one specified in the test file. I just added the entry in the test file to account for the new property and fix the test.

Reply:
I don't know if this answers the question, but I am not sure if the property should or shouldn't be serialized.